packages/orchestrator-ui-components/src/components/WfoSummary/WfoSummaryCards.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { FC } from 'react';
+
+import {
+    EuiFlexGrid,
+    EuiFlexItem,
+    EuiSpacer,
+    useCurrentEuiBreakpoint,
+} from '@elastic/eui';
+
+import { SummaryCardListItem } from '@/components/WfoSummary/';
+import {
+    WfoSummaryCardHeader,
+    WfoSummaryCardHeaderProps,
+} from '@/components/WfoSummary/WfoSummaryCardHeader';
+import { useOrchestratorTheme } from '@/hooks';
+
+import {
+    SummaryCardButtonConfig,
+    WfoSummaryCardList,
+} from './WfoSummaryCardList';
+import { getNumberOfColumns } from './getNumberOfColumns';
+
+export enum SummaryCardStatus {
+    Success = 'Success',
+    Error = 'Error',
+    Neutral = 'Neutral',
+}
+
+export type SummaryCard = {
+    headerTitle: string;
```

Comment:
Add a loading and or an error state? So you can pass that in and present a loading state

Reply:
Added a skeleton for loading state